### ISSUE TITLE
Why is it an error to run `ember new` from an empty directory (with no package.json)?

### ISSUE THREAD

**Issue by alanhogan** [emberjs/ember.js#11693]
Howdy,

I’ve been following Ember for a while but only just tried to make my first ember app. After running the lines at the top of emberjs.com, I experienced an error.

### What I did

(I’m on a very new MacBook Pro)

1. Switched my shell to `zsh`

2. Created a new empty folder and `cd`'d into it

3. Installed node

4. Ran `sudo npm install-g ember-cli`

5. Run `ember new keysplitapp`

6. Got this error:

    ```sh
    ENOENT, open '/Users/ahogan/Repos/personal/keysplit-ui/package.json'
Error: ENOENT, open '/Users/ahogan/Repos/personal/keysplit-ui/package.json'
    at Error (native)
    ```

I'll include the full output at the end of this issue.

I don’t understand what went wrong — did I miss a step? Is there an assumption I have to already have a package.json? This doesn’t seem to be mentioned on the Getting Started page.

Full paste from console:

```sh
Repos/personal% cd keysplit-ui
personal/keysplit-ui% ember new keysplitapp
version: 0.2.7
0.2.7

Could not find watchman, falling back to NodeWatcher for file system events.
Visit http://www.ember-cli.com/#watchman for more info.
installing
  create .bowerrc
  create .editorconfig
  create .ember-cli
  create .jshintrc
  create .travis.yml
  create .watchmanconfig
  create Brocfile.js
  create README.md
  create app/app.js
  create app/components/.gitkeep
  create app/controllers/.gitkeep
  create app/helpers/.gitkeep
  create app/index.html
  create app/models/.gitkeep
  create app/router.js
  create app/routes/.gitkeep
  create app/styles/app.css
  create app/templates/application.hbs
  create app/templates/components/.gitkeep
  create bower.json
  create config/environment.js
  create .gitignore
  create package.json
  create public/crossdomain.xml
  create public/robots.txt
  create testem.json
  create tests/.jshintrc
  create tests/helpers/resolver.js
  create tests/helpers/start-app.js
  create tests/index.html
  create tests/test-helper.js
  create tests/unit/.gitkeep
  create vendor/.gitkeep
ENOENT, open '/Users/ahogan/Repos/personal/keysplit-ui/package.json'
Error: ENOENT, open '/Users/ahogan/Repos/personal/keysplit-ui/package.json'
    at Error (native)
```

**Comment by stefanpenner** [comment]
FYI:

wrong repo: https://github.com/ember-cli/ember-cli
related issue:

TL;DR some race condition, that only happens the "first time".

**Comment by alanhogan** [comment]
Thanks @stefanpenner

**Comment by stefanpenner** [comment]
@alanhogan sorry for this lame bug
